workflows/proteomics/clinicalmp/clinicalmp-data-interpretation/WF5_Data_Interpretation_Worklow.ga

```diff
@@ -0,0 +1,645 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Clinical Metaproteomics 5: Data Interpretation",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "GalaxyP"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "WF5_Data_Interpretation_Worklow",
```

Comment:
```suggestion
    "name": "Clinical Metaproteomics Data Interpretation,
```
This is the name users see in the workflow list and when they go to run a workflow. Does this make sense as a workflow name ?